.vscode/launch.json

```diff
@@ -0,0 +1,34 @@
+{
+  // Launch configuration to build and launch Kratos
```

Comment:
does this parse?

Reply:
yep, VS Code launch configs allow comments.